util/CMakeLists.txt

```diff
@@ -345,6 +345,11 @@ add_library(
     cgroup_parser.cc
 )
 
+target_link_libraries(
```

Comment:
Notice how for other libraries in this cmakefile we don't leave an empty line between individual statements for the same library.

Reply:
fixed